src/components/pages/TAIR/styles/design.css

```
@@ -0,0 +1,402 @@
+    display: block; 
+    background: var(--primary-red);
+    color: white;
+    padding: 3px 8px;
+    border-radius: 4px; 
+    font-size: 0.9em; 
+    font-weight: bold;
+    margin-bottom: 5px; 
+    margin-right: 0; 
+    width: fit-content;
+    margin-left: auto;
+    margin-right: auto;
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

margin-right is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)